examples/forbid-anonymous-operations/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forbid-anonymous-operations"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.54"
+apollo-router = { path = "../../apollo-router" }
+apollo-router-core = { path = "../../apollo-router-core" }
+futures = "0.3.21"
+tokio = { version = "1.16.1", features = ["full"] }
+tracing = "0.1.31"
+tokio-util = { version = "0.7.0", features = ["net", "codec"] }
+tracing-subscriber = "0.3.9"
+tower = { version = "0.4.12", features = ["full"] }
+serde_json = "1.0.79"
+serde = "1.0.136"
+serde_json_bytes = "0.2.0"
+apollo-parser = "0.2.3"
+http = "0.2.6"
```

Comment:
Same comment as above. We should definitely be consistent.

Reply:
fixed in def7ea6a